.gitignore

```diff
@@ -1,20 +1,20 @@
-/build/
-!/build/cache/.gitkeep
+!/.tools/.gitkeep
```

Comment:
It looks like the order is platform dependent, maybe it would be wise to run that command in a docker container to avoid this issue

Reply:
Or locale dependent. `LC_ALL=C sort` may help.

Reply:
Will try in a separate PR and remove the changes made to the gitignore here

Reply:
Think you can merge with master.